src/ledger/ledger-db/k8s/overlays/development/kustomization.yaml

```diff
@@ -14,8 +14,11 @@
 
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
+commonLabels:
+  tier: db
 resources:
   - ../../base
   - ledger-db.yaml
 components:
-  - ../../../../../components/development
\ No newline at end of file
+  - ../../../../components/ledger
```

Comment:
This is already set in the base: https://github.com/GoogleCloudPlatform/bank-of-anthos/blob/main/src/ledger/ledger-db/k8s/base/kustomization.yaml#L22

Isn't it a duplicate here in the overlay?